topics/single-cell/tutorials/scrna_batch_correction/tutorial.md

```diff
@@ -0,0 +1,529 @@
+
+We are using a single cell dataset of human Peripheral Blood Mononuclear Cells (PBMCs) that was also used in Seurat's [Integrative analysis in Seurat v5](https://satijalab.org/seurat/articles/seurat5_integration) tutorial. The original study compared the results from seven different single cell and single nuclear techniques {% cite Ding2020 %}.
+
+Let's take a look at our data before we begin the analysis to see whether we might need to perform batch correction or integration.
+
+<div class='Scanpy' markdown='1'>
+
+SCANPY INSPECT
+
+</div>
+
+<div class='Seurat' markdown='1'>
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Seurat Data Management](toolshed.g2.bx.psu.edu/repos/iuc/seurat_data/seurat_data/5.0+galaxy0) %} with the following parameters:
+>    - *"Method used"*: `Inspect Seurat Object`
+>        - *"Display information about"*: `General`
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many cells and genes are in this dataset?
+> 2. How many layers are in this dataset?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. If we click on the {% icon galaxy-eye %} of the new output in our history, we can see that this dataset contains information about the expression of 33,694 features (genes) in 10,434 samples (cells).
+> > 2. Data in SeuratObjects are stored in layers. In this case, we only have one layer called `counts`. The `counts` layer is the raw data that we'll be using in this tutorial.
+> >
+> {: .solution}
+>
+{: .question}
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Seurat Data Management](toolshed.g2.bx.psu.edu/repos/iuc/seurat_data/seurat_data/5.0+galaxy0) %} with the following parameters:
+>    - *"Method used"*: `Inspect Seurat Object`
+>        - *"Display information about"*: `Cell Metadata`
+>
+{: .hands_on}
+
+</div>
+
+> <question-title></question-title>
+>
+> 1. What does the `Method` column represent in the cell metadata?
+> 2. Do you think batch correction or integration is needed for this analysis?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The dataset that we're using comes from a study that compared different single cell techniques. The `Method` column tells us which technique was used on each cell.
+> > 2. Each experimental technique can be considered as its own experimental batch. Each of these batches was processed independently, which by itself can be enough to require batch correction, even if the same experimental protocol is used - batches can vary simply because they were processed at different times or by different people in the same lab! In this case, we have an even stronger reason to believe that these batches will differ - we know that each batch was produced using a different technique, so it seems likely that we'll need to perform batch correction. We would consider this to be batch correction rather than integration because these data all came from the same original study.
```

Comment:
```suggestion
> > 2. Each experimental technique can be considered as its own experimental batch. Each of these batches was processed independently, which by itself can be enough to require batch correction, even if the same experimental protocol is used. Batches can vary simply because they were processed at different times or by different people in the same lab! In this case, we have an even stronger reason to believe that these batches will differ - we know that each batch was produced using a different technique. It seems likely that we'll need to perform batch correction. We would consider this to be batch correction rather than integration because these data all came from the same original study.
```